### PR TITLE
Redesign Footer

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
Goal:

footer

## Todo:

- [x] Add blocks, content and basic styling
- [x] Make styling good
- [x] Connect with Mailchimp 
- [x] Real content
  - [x] NL
  - [x] DE
  - [x] EN
- [x] Setup Mailchimp
  - [x] Thanks messages in all 3 locales
  - [x] Setup Dutch Newsletter
  - [x] Setup German Newletter
  - [x] Setup English Newsletter
- [x] Make it sticky
- [x] Test in all browsers

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
OK, basic implementation with MailChimp for newsletter signups works. 

Tomorrow tryin to get the rest of the todo's done.

**Comment by RonaldDijkstra** [comment]
Ready for review.

**Comment by RonaldDijkstra** [comment]
Tested in: IE10+, Edge, Firefox, Chrome, Safari, Mobile chrome, mobile safari.

File: source/stylesheets/_footer.scss [hunk 1]
```diff
@@ -1,92 +1,219 @@
 .footer {
   background-color: $dark-gray;
-  clear: both;
-  color: $medium-gray;
+  color: $testimonial-gray;
```
Comment (sn3p) [inline source/stylesheets/_footer.scss]: `$footer-gray`?

And seems like we use `$footer-gray` in `_testimonials.scss`.

File: source/stylesheets/_jobs.scss [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
-  a {
-    color: $blog-link-color;
-    text-decoration: none;
-
-    &:active,
-    &:focus,
-    &:hover {
-      color: darken($blog-link-color, 15%);
+  .main {
+    a {
```
[the remark this answers is not in the document]
Comment (RonaldDijkstra) [reply source/stylesheets/_jobs.scss]: Will do later.

File: source/stylesheets/_footer.scss (hunk 1 above)
Comment (RonaldDijkstra) [reply source/stylesheets/_footer.scss]: I'll refactor all color things in another PR, as part of #370

**Pull request merged**